src/app/components/App.jsx

```diff
@@ -30,11 +30,15 @@ import PlaylistsProvider from './Providers/PlaylistsProvider';
 import AlbumPage from './Routes/Album/AlbumPage';
 
 function App() {
-  useEffect(() => {
-    if (process.env.NODE_ENV !== 'development') {
-      document.addEventListener('contextmenu', event => event.preventDefault());
-    }
-  }, []);
+  if (process.env.NODE_ENV !== 'development') {
```

Comment:
https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
Already changed.